scipy/stats/stats.py

```diff
@@ -101,6 +101,7 @@
 
    pearsonr
    fisher_exact
+   barnard_exact
```

Comment:
I'm not sure where this renders, if anywhere.

Reply:
I don't think it's anywhere. Can that be tackled separately ?

Reply:
Of course. I just mean that this wasn't rendering the docs for `barnard_exact`. I added `barnard_exact` to `__init__.py` so they would render.